compiler/p/codegen/ControlFlowEvaluator.cpp

```diff
@@ -3307,23 +3300,25 @@ static void lookupScheme4(TR::Node *node, TR::CodeGenerator *cg)
       int32_t offset = TR_PPCTableOfConstants::allocateChunk(1, cg);
 
       if (offset != PTOC_FULL_INDEX)
-	 {
+         {
          offset *= 8;
          TR_PPCTableOfConstants::setTOCSlot(offset, address);
          if (offset<LOWER_IMMED||offset>UPPER_IMMED)
-	    {
+            {
             generateTrg1Src1ImmInstruction(cg, TR::InstOpCode::addis, node, pivotRegister, cg->getTOCBaseRegister(), cg->hiValue(offset));
             generateTrg1MemInstruction(cg,TR::InstOpCode::Op_load, node, addrRegister, new (cg->trHeapMemory()) TR::MemoryReference(pivotRegister, LO_VALUE(offset), 8, cg));
-	    }
+            }
          else
-	    {
+            {
             generateTrg1MemInstruction(cg,TR::InstOpCode::Op_load, node, addrRegister, new (cg->trHeapMemory()) TR::MemoryReference(cg->getTOCBaseRegister(), offset, 8, cg));
-	    }
-	 }
+            }
+         }
       else
-	 {
-         loadAddressConstant(cg, node, address, addrRegister);
-	 }
+         {
+         // the loadAddress routine here will eventually map down to TR_FixedSequence2 relo type. Might need to generate a query here that moves this JITServer
+         // check downstream.
+         loadAddressConstant(cg, node, address, addrRegister, cg->comp()->isOutOfProcessCompilation() || cg->comp()->isOutOfProcessCompilation());
```

Comment:
Likewise here.

Reply:
See here for fix: 50cbd5359f037696490e20cd2c7a9b2d82caf747